code/game/objects/items/storage/fancy.dm

```diff
@@ -125,6 +125,10 @@
 	icon_type = "cigarette"
 	spawn_type = /obj/item/clothing/mask/cigarette/space_cigarette
 
+/obj/item/storage/fancy/cigarettes/examine(mob/user)
+	..()
+	to_chat(user, "<span class='notice'>Alt-click to extract contents.</span>")

```

Comment:
Don't know where to comment this, but alt-clicking didn't work on the cigarette-packs except if it had a lighter in it, then it would be taken out, no idea where the problem was. Should a separate issue be made or does someone have an idea if it's an easy fix that could be done in this PR?

Reply:
just go to the code RIGHT below this and change it to get the last item in its contents(target using .len).

its just so that you can get a lighter out if your doing the attack mouth trick

Reply:
Alright, cigarette pack alt-clicking fixed.